src/screens/OtherProfile/index.js

```diff
@@ -289,8 +297,8 @@ const OtherProfile = () => {
         <React.Fragment>
           {__renderFollowingButton()}
           <GlobalButton onPress={onCreateChat}>
-            <View style={styles.btnMsg}>
-              <EnveloveBlueIcon width={20} height={20} fill={colors.bondi_blue} />
+            <View style={styles.btnMsg(isAnonimity)}>
+              <EnvelopeIcon color={isAnonimity ? COLORS.anon_primary : COLORS.signed_primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `EnvelopeIcon` component is being used here with a dynamic color prop. This is a good practice as it allows the icon color to be easily customized.



<!-- This is an auto-generated comment by OSS CodeRabbit -->